### ISSUE TITLE
Provide an command tool to transform package namespace in archive file

### ISSUE THREAD

**Issue by quaff** [raphw/byte-buddy#990]
As you know, Jakarta EE 9 is not backward-compatible with Jakarta EE 8 or Java EE 8, I hope there is a tool will replace all `javax.*` to `jakarta.*` in WAR/EAR files, I'm here since byte-buddy is bytecode expert, not sure this is the right place to post it, feel free to close it.

**Comment by quaff** [comment]
https://github.com/apache/tomcat-jakartaee-migration

**Comment by raphw** [comment]
It would be a good idea to have a generic renaming plugin in Byte Buddy, I'll reopen this issue as a reminder for this feature.

**Comment by quaff** [comment]
> It would be a good idea to have a generic renaming plugin in Byte Buddy, I'll reopen this issue as a reminder for this feature.

Great👍

**Comment by be-hase** [comment]
@raphw 

Due to spring boot3, there will be more and more products that need to be changed from `javax.*` to `jakarta.*`.
Do you plan to work on this issue?

**Comment by raphw** [comment]
Good point. I just added a `RenamingPlugin` that can easily be integrated. I will do a release some time soon where it is included. See the javadoc for the `RenamingPlugin` to see how it can be applied.

**Comment by be-hase** [comment]
Wow. I went to bed and woke up to find it implemented... hahaha
https://github.com/raphw/byte-buddy/commit/0f6ca3ad7c1acb63c553b7dc3539c43b72abfc4b

Thanks !
